elasticdl/python/master/embedding_service.py

```diff
@@ -310,26 +197,19 @@ def lookup_embedding(
                 unknown_keys_idx.append(index)
         return embedding_vectors_ndarray, unknown_keys_idx
 
-    @staticmethod
     def update_embedding(
-        keys=[],
-        embedding_vectors=[],
-        embedding_service_endpoint=None,
-        set_if_not_exist=False,
+        self, keys=[], embedding_vectors=[], set_if_not_exist=False
```

Comment:
Default values of function arguments should be `None` or plain types.

Reply:
Done